paimon-core/src/main/java/org/apache/paimon/utils/FileStorePathFactory.java

```diff
@@ -37,6 +37,17 @@
 @ThreadSafe
 public class FileStorePathFactory {
 
+    public static final String MANIFEST_PATH = "manifest";
+    public static final String MANIFEST_PREFIX = "manifest-";
+    public static final String MANIFEST_LIST_PREFIX = "manifest-list-";
+    public static final String INDEX_MANIFEST_PREFIX = "index-manifest-";
+
+    public static final String INDEX_PATH = "index";
+    public static final String INDEX_PREFIX = "index-";
+
+    public static final String STATISTICS_PATH = "statistics";
+    public static final String STATISTICS_PREFIX = "stat-";
+
```

Comment:
private

Reply:
I think both public and private are fine, and the BUCKET_PATH_PREFIX is also public.